src/main/java/net/fabricmc/loom/configuration/providers/LaunchProvider.java

```diff
@@ -87,7 +87,13 @@ public void provide(DependencyInfo dependency, Consumer<Runnable> postPopulation
 	}
 
 	private File getLog4jConfigFile() {
-		return new File(getExtension().getDevLauncherConfig().getParentFile(), "log4j.xml");
+		return getExtension().getDefaultLog4jConfigFile();
+	}
+
+	private String getAllLog4JConfigFiles() {
+		return getExtension().getLog4jConfigs().getFiles().stream()
+				.map(File::getAbsolutePath)
+				.collect(Collectors.joining(","));
```

Comment:
Huh, TIL you could have multiple log4j config files! This might actaully turn out to be quite useful for some stuff I wanted to do with the vanilla logger. 🤔 

https://logging.apache.org/log4j/2.x/manual/configuration.html#CompositeConfiguration

Nice 👍